libs/components/tabs/src/lib/modules/tabs/tab.component.ts

```diff
@@ -87,28 +88,38 @@ export class SkyTabComponent implements OnChanges, OnDestroy {
    * @required
    */
   @Input()
-  public tabHeading: string;
+  public set tabHeading(value: string | undefined) {
+    this.#_tabHeading = value;
+
+    if (!this.#permalinkValue) {
```

Comment:
I'd expect this logic to be identical to the logic in the `permalinkValue` setter. I'd move that logic into a function that gets the value of `permalinkValue` and `tabHeading` and sets a `permalinkValueOrDefault` public property that is then referenced in the template. That will also let you go back to using the `#_permalinkValue` convention since it'll only be referenced in the `permalinkValue` accessors. Alternatively you could create a pure pipe that accepts `permalinkValue` and `tabHeading` and returns the value to use in the template. That would be a cleaner solution but would result in more code.